### PR TITLE
Adding ADR for using maplibre to repo (see issue #299)

### PR DESCRIPTION
**Pull request by NatHillardUSDS**

### CODE REVIEW

File: docs/decisions/0007-maplibre.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+- Mapbox GL JS 1.13.0
+- OpenLayers 6.4.3
+
+## Decision Outcome
+
+Chosen option: [MapLibre 1.15.0](https://github.com/maplibre/maplibre-gl-js), a free and open-source alternative to Mapbox with good performance and standard map styling cabilities, and continued development lifecycle.
+
+### Positive Consequences
+
+- MapLibre is a free and open source alternative
```
Comment (TomNUSDS) [inline docs/decisions/0007-maplibre.md:27]: `alternative` work is not needed here, it's used in the section before

File: docs/decisions/0007-maplibre.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+## Considered Options
+
+- MapLibre GL JS 1.15.0
+- Mapbox GL JS 2.3.1
+- Mapbox GL JS 1.13.0
+- OpenLayers 6.4.3
+
+## Decision Outcome
+
+Chosen option: [MapLibre 1.15.0](https://github.com/maplibre/maplibre-gl-js), a free and open-source alternative to Mapbox with good performance and standard map styling cabilities, and continued development lifecycle.
```
Comment (TomNUSDS) [inline docs/decisions/0007-maplibre.md:23]: It's not just an alternative to Mapbox, right? Maybe we can reword this sentence to be more generic?

File: docs/decisions/0007-maplibre.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+### Positive Consequences
+
+- MapLibre is a free and open source alternative
+- Performance is preferable to OpenLayers, as seen in this video: [MapLibre Performance](https://user-images.githubusercontent.com/72811320/124788369-446a6f80-df17-11eb-9dec-6635f8bfc3cb.mp4)
```
Comment (TomNUSDS) [inline docs/decisions/0007-maplibre.md:28]: We should note that this url may change (or go away). Might be better to upload and forget about it on youtube?

File: docs/decisions/0007-maplibre.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+- There might be other incompatibilities with mapbox and/or backwards incompatibilities as the Maplibre framework evolves
+  - Mitigation: we will follow the progress of MapLibre as it evolves
+
+## Pros and Cons of the Options
+
+### Mapbox-GL JS 2.3.1
+
+The latest released version of [Mapbox](https://docs.mapbox.com/mapbox-gl-js/api/)
+
+- Good: performance is quite good and improving
```
Comment (TomNUSDS) [inline docs/decisions/0007-maplibre.md:46]: "Good/Bad" -> "Pro/Con"

**Review by TomNUSDS — APPROVED** [review]
Made some suggestions, but otherwise approved. ❤️ documentation

File: docs/decisions/0007-maplibre.md (hunk 3 above)
Comment (NatHillardUSDS) [reply docs/decisions/0007-maplibre.md:28]: ah true, I had added the video to the repo but hadn't updated the link

**Review by TomNUSDS — APPROVED** [review]
Thanks! LGTM!

**Pull request merged**